torch/csrc/jit/tensorexpr/external_functions.cpp

```diff
@@ -41,6 +47,88 @@ std::vector<at::Tensor> constructTensors(
   return tensors;
 }
 
+std::vector<ideep::tensor> constructItensors(
+    int64_t bufs_num,
+    void** buf_data,
+    int64_t* buf_ranks,
+    int64_t* buf_dims,
+    int8_t* buf_dtypes) {
+
+
+  std::vector<void*> buf_data_vec;
+  std::vector<std::vector<int64_t>> buf_dims_vec;
+  std::vector<c10::ScalarType> buf_dtypes_vec;
+
+  int64_t buf_dims_idx = 0;
+  for (int64_t i = 0; i < bufs_num; i++) {
+    buf_data_vec.push_back(buf_data[i]);
+    buf_dims_vec.emplace_back();
+    for (int64_t dim = 0; dim < buf_ranks[i]; dim++) {
+      buf_dims_vec[i].push_back(buf_dims[buf_dims_idx++]);
+    }
+    buf_dtypes_vec.push_back(static_cast<c10::ScalarType>(buf_dtypes[i]));
+  }
+
+  // TODO: support more types
+  std::unordered_map<c10::ScalarType, ideep::data_type> c102it {{c10::ScalarType::Float, ideep::data_type::f32}};
+  auto tag = ideep::format_tag::undef;
+  // TODO: support more layouts         0            1             2    3              4  
+  std::vector<ideep::format_tag> tags {tag, ideep::format_tag::a, tag, tag, ideep::format_tag::nchw};
+
+  std::vector<ideep::tensor> tensors;
+  for (size_t i = 0; i < buf_data_vec.size(); i++) {
+    tensors.emplace_back(ideep::tensor(buf_dims_vec[i], c102it.at(buf_dtypes_vec[i]), tags[buf_dims_vec[i].size()], buf_data_vec[i]));
+  }
+  return tensors;
+}
+
+void nnc_aten_conv2d_out(
+  int64_t bufs_num,
+  void** buf_data,
+  int64_t* buf_ranks,
+  int64_t* buf_dims,
+  int8_t* buf_dtypes,
+  int64_t args_num,
+  int64_t* extra_args) {
+
+  if (args_num > 0) {
```

Comment:
Could we construct default bias and parameters for `args_num == 0` case? Otherwise, the perf might be surprisingly different when they are omitted.